instagram/css/main.css

```
@@ -0,0 +1,231 @@
+header {
+    display: flex;
+}
+
+main {
+    background-color: #fafafa;
+    padding-top: 54px;
+
+}
+
+h2 {
+    font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", sans-serif;
+    font-size: 28px;
+    line-height: 32px;
+    margin: -5px 10px -6px;
+}
+
+nav {
+    position: fixed;
+
+}
+
+.nav-container {
+    display: flex;
+    border-bottom: 1px solid rgb(219, 219, 219);
+    height: 54px;
+    width: 100%;
+    padding: 0 10px;
+    background-color: #ffffff;
+}
+
+.nav-item-container {
+    display: flex;
+    margin-right: auto;
+    flex: none;
+    padding: 10px 5px;
+}
+
+.instagram-icon {
+    padding: 5px 3px;
+}
+
+.instagram-logo {
+    display: flex;
+    padding-top: 5px;
+    max-width: 40%;
+
+}
+
+.nav-item-gnb-container {
+    display: flex;
+    margin-left: auto;
+    flex: none;
+    padding: 10px 10px;
+}
+
+.nav-item-gnb {
+    display: flex;
+    padding: 5px 10px;
+}
+
+.profile-img-container {
+    height: 77px;
+    width: 77px;
+    padding: 30px 10px 10px 15px;
+    margin-right: 0;
+}
+
+.profile-img {
+    display: flex;
+    background-color: #fafafa;
+    border: 0.5px solid #c8c8c8;
+    border-radius: 50%;
+    height: 100%;
+    margin: 0 auto;
+    overflow: hidden;
+    width: 100%;
+}
+
+.profile-edit-container {
+    padding: 40px 10px 10px 10px;
+
+}
+
+.id {
+    display: flex;
+}
+
+.edit-btn {
+    display: flex;
+    padding: 0 10px 10px 1px;
+    margin-right: 1px;
+}
+
+.profile-edit-btn {
+    display: flex;
+    padding-top: 10px;
+    padding-left: 10px;
+}
+
+.profile-edit-btn button {
```

자손 선택자보단 자식 선택자를 사용하는 것을 권장드려요 :) [여기](https://amaze9001.tistory.com/45)를 참고하면 좋겠네요!

```suggestion
.profile-edit-btn > button {
```